vividus-plugin-html/src/main/java/org/vividus/util/HtmlUtils.java

```diff
@@ -26,14 +28,33 @@ private HtmlUtils()
     {
     }
 
+    public static Elements findElements(String html, LocatorType locatorType, String locator)
```

Comment:
```suggestion
    public static Elements getElements(String html, LocatorType locatorType, String locator)
```

I think we should follow common naming convention for all methods in the class